.github/workflows/manual-deploy-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs deploy to Netlify
+on:
+  workflow_dispatch:
+    inputs:
+      docsVersion:
+        description: "Version of the Docs to release. Must be in format: vX eg.: v6 v7 ..."
+        required: true
+        default: "v7"
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy docs
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js environment.
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 10
+      - name: Install packages.
+        run: yarn install
+      - name: Set up project.
+        run: yarn bootstrap
+      - name: Build docs-app.
+        run: yarn build:docs
+      - name: Deploy docs-app to netlify
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
```

Comment:
Shouldn't these be added to `env.example`?

Reply:
These env vars are used only in the CI environment.
If we would like to deploy to netlify from a local environment I'd suggest using the [command line authentication](https://docs.netlify.com/cli/get-started/#authentication) which is much more user friendly than obtaining the auth token from netlify, then assigning it to an env var. (everyone would have to do this, since the auth token is exposed exactly 1 time, upon creating it)

Reply:
If you would like to see it in action, I'm more than happy to show it.